listling/listling.py

```diff
@@ -114,6 +115,27 @@
                                      (52.497561, 13.430773))
             }
         ]
+    ),
+    'presentation': (
+        'Brief résumé of my work experience',
+        'Max Mustermann \n\n  *You will find in the menu the button to activate the presentation mode*',
+        [
+            {
+                'title': 'My job as a programmer',
+                'text': 'I wrote a 2D MMORPG for Star Wars fans.',
+                'resource': 'https://c.wallhere.com/photos/e4/80/4096x2160_px_artwork_STAR_WARS_Battlefront_GAME_Star_Wars_Battlefront_video_games-1102412.jpg!d'
```

Comment:
Found a bug here :sweat_smile: You cannot directly link to the image - it is now blocked by the hoster.

I think we can do one of two things:

A)
1. Add an image (must have an open source license) to the directory `listling/res`
1. Set `resource` to the name of the image
1. In `Lists.create_example()`, if `resource` is not a URL, `open()` it and store it with `app.files.write()`. The result is a URL we can use.

B) Drop images for now :)

In any case, I would only add an image to one item, for better clarity :) .